src/components/Text/types.ts

```diff
@@ -11,6 +11,9 @@ export enum TextVariants {
   body1 = 'body1',
   body2 = 'body2',
   body3 = 'body3',
+  string1 = 'string1',
```

Comment:
The naming is confusing here. What is the use case for a string in this context?

Reply:
tbh I didn't ask them, this is what design provided, looks like it's gonna be used for Inputs and etc